app/models/Talk.php

```diff
@@ -47,8 +43,23 @@ public function getRevisionsAttribute()
         return $this->revisions()->getQuery()->orderBy('created_at', 'desc')->get();
     }
 
+    public function isArchived()
+    {
+        return $this->attributes['is_archived'];
```

Comment:
double check that this is always going to return a boolean; i don't know whether it will get cast on its way into `attributes`. Could you just do `$this->is_archived`